rom1:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Velocity Controllers ---------------------------------------
  front_right_wheel_controller:
    type: velocity_controllers/JointVelocityController
    joint: front_right_wheel_joint
    pid: {p: 100.0, i: 10.0, d:  0.1}
  front_left_wheel_controller:
    type: velocity_controllers/JointVelocityController
    joint: front_left_wheel_joint
    pid: {p: 100.0, i: 10.0, d:  0.1}
  back_right_wheel_controller:
    type: velocity_controllers/JointVelocityController
    joint: back_right_wheel_joint
    pid: {p: 100.0, i: 10.0, d:  0.1}
  back_left_wheel_controller:
    type: velocity_controllers/JointVelocityController
    joint: back_left_wheel_joint
    pid: {p: 100.0, i: 10.0, d:  0.1} 

  # gazebo_ros_control: 
  #   pid_gains:
  #     front_right_wheel_joint:
  #       p: 100.0
  #       i: 0.01
  #       d: 10.0
  #     front_left_wheel_joint:
  #       p: 100.0
  #       i: 0.01
  #       d: 10.0
  #     back_right_wheel_joint:
  #       p: 100.0
  #       i: 0.01
  #       d: 10.0
  #     back_left_wheel_joint:
  #       p: 100.0
  #       i: 0.01
  #       d: 10.0
      
      